pytorch/inference/docker/1.10/py3/sdk1.17.1/Dockerfile.neuron

```diff
@@ -106,7 +106,7 @@ RUN pip install --no-cache-dir -U \
     boto3
 
 RUN pip install neuron-cc[tensorflow] --extra-index-url https://pip.repos.neuron.amazonaws.com \
- && pip install "torch-neuron>=1.10.1,<1.10.2" --extra-index-url https://pip.repos.neuron.amazonaws.com \
+ && pip install "torch-neuron==1.10.1.2.1.7.0" --extra-index-url https://pip.repos.neuron.amazonaws.com \
```

Comment:
why are we pinning specific torch-neuron here? apologies if i missed something in previous correspondence

Reply:
I was debugging a different Neuron failure and @aws-vrnatham explained that this is needed since Neuron SDK 1.19.0 was released which meant that not pinning the version would use an updated version which failed our CI.